src/Telemetry/Samples/FSharp/Program.fs

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+open System
+open Microsoft.Quantum.Telemetry
+
+type SampleUnionType =
+    | SampleEnumValue1
+    | SampleEnumValue2
+
+type ExecutionCompleted =
+    {
+        SampleDateTime: DateTime
+        SampleString: string
+        SampleBool: bool
+        SampleEnum: SampleUnionType
+        [<PiiData>]
+        SamplePII: string
+        [<SerializeJson>]
+        SampleArray: string []
+        SampleTimeSpan: TimeSpan
+        SampleInt: int
+        SampleGuid: Guid
+        [<SerializeJson>]
+        SampleObjectToBeSerialized: Map<string, string>
+        SampleObjectToBeIgnored: obj
+        SampleException: Exception
+        SampleNullableWithValue: Nullable<int>
+        SampleNullableWithNull: Nullable<int>
```

Comment:
System.Nullable generally isn't used in F# unless it's specifically for C# interop. More natural is to use `int option`. It looks like the telemetry library has special support for ignoring nulls; does it also support F# options?

Reply:
Thanks, Sarah. I'll change it to use F# options and make sure it works well.

Reply:
Added support to use F# Options and updated the F# Sample.